src/nxdoc/nuxeo-server/nuxeo-server-release-notes.md

```diff
@@ -1725,49 +1726,49 @@ You do not need to depend on `nuxeo-dam` package anymore on your Nuxeo package,
 
 <i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-28626](https://jira.nuxeo.com/browse/NXP-28626)
 
-### Nuxeo Wizard
+### Nuxeo Wizard {{> tag 'dev'}} {{> tag 'admin'}}
 
 The Nuxeo Wizard has been removed.
 
 <i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXP-28621](https://jira.nuxeo.com/browse/NXP-28621)
 
-### Marklogic Connector
+### Marklogic Connector {{> tag 'dev'}}
```

Comment:
Add basically admin for all architecture components / connectors which are not supported